### PR TITLE
Add Spring

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
Spring

**Issue:** #5750
**Alexa rank:** [~24.6k](https://www.alexa.com/siteinfo/teespring.com)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Icon from SVG in [website](https://www.spri.ng/) header. Open to suggestions on the `slug`.

### CODE REVIEW

**Review by adamrusted — APPROVED** [review]
This looks good to me - will you be updating Teespring in a separate PR?
If so, this can be merged straight in.

**Comment by PeterShaggyNoble** [comment]
Yeah, @adamrusted, Teespring can be updated in a separate PR to avoid it blocking this one. Will merge this in once the checks have passed.

**Pull request merged**
